docs/articles/expensify-classic/getting-started/Plan-Types.md

```diff
@@ -2,4 +2,31 @@
 title: Plan-Types
 description: Plan-Types
```

Comment:
So I think (?) these are user facing, so you'll probably want `Plan Types` for the title (without the hyphen) and then a little bit more detail on the description - e.g. `Learn about the different plans you can sign up for on Expensify` but, you know, better copy.